elyra/util/git.py

```diff
@@ -0,0 +1,50 @@
+#
+# Copyright 2018-2021 Elyra Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from github import Github, GithubException
+from traitlets.config import LoggingConfigurable
+
+
+class GithubClient(LoggingConfigurable):
+
+    def __init__(self,
```

Comment:
Can you please add a github server property that defaults to `https://api.github.com`? This is the implicit default in https://pygithub.readthedocs.io/en/latest/github.html#github.MainClass.Github.  See https://github.com/elyra-ai/elyra/issues/1313 for context.